fedimint-bitcoind/src/lib.rs

```diff
@@ -17,6 +18,13 @@ pub mod bitcoincore_rpc;
 /// Functions may panic if if the bitcoind node is not reachable.
 #[async_trait]
 pub trait IBitcoindRpc: Debug + Send + Sync {
+    /// `true` if it's real-bitcoin (not electrum) backend and thus supports `get_block` call
+    ///
+    /// This is a bit of a workaround to support electrum.
+    fn is_bitcoind(&self) -> bool {
+        true
+    }
```

Comment:
Could we make this a function that returns an enum instead that tells us which tx confirmation query strategy to use? That would be more general. E.g. an esplora client might act like bitcoind, but certainly isn't.

Reply:
Good idea.